### PR TITLE
add tcmarkers

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/tcmarkers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/t/tcmarkers/tcmarkers-{{ version }}.tar.gz
+  sha256: 2e31ee5bae5462b6d23b8902af748fc419792512c5893deb7d7e25ceb16a6385
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (BastianZim) [inline recipes/tcmarkers/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```
The rest is taken are by the conda-forge infrastructure.

File: recipes/tcmarkers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - setuptools_scm
+    - setuptools_scm_git_archive
+  run:
+    - python >=3.7
+    - matplotlib-base
+    - numpy
+
+test:
+  imports:
+    - tcmarkers
```
Comment (BastianZim) [inline recipes/tcmarkers/meta.yaml:29]:
```suggestion
test:
  imports:
    - tcmarkers
  commands:
    - pip check
  requires:
    - pip
```
This makes sure that all packages are there.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

**Comment by raybellwaves** [comment]
> Just a couple of small things, then this is ready to go 🚀

Thanks i'll update my generic template with this feedback when I add a repo

**Comment by BastianZim** [comment]
> Thanks i'll update my generic template with this feedback when I add a repo

I just use Grayskull for simple python recipes, it's the fastest and easiest way.

**Review by BastianZim — APPROVED** [review]

**Comment by BastianZim** [comment]
Please ping me when the CI passes and I'll merge it.

**Comment by raybellwaves** [comment]
Friendly ping @BastianZim

**Pull request merged**